src/actionCable/actionCable.js

```diff
@@ -0,0 +1,22 @@
+import { Platform } from 'react-native';
+import ActionCable from 'actioncable';
+
+ActionCable.getConfig = () => null;
+
+ActionCable.createWebSocketURL = url => url.replace(/^http/, 'ws');
+
+const oldOpen = ActionCable.Connection.prototype.open;
+ActionCable.Connection.prototype.open = function open() {
+  const result = oldOpen.apply(this);
+  this.webSocket.protocol = 'actioncable-v1-json';
+  return result;
+};
+
+if (Platform.OS === 'ios' || Platform.OS === 'android') {
```

Comment:
Is this needed?

Reply:
Oh, I was going to remove that and forgot it. Thanks!